src/seq.rs

```diff
@@ -369,6 +371,13 @@ impl PortInfo {
         unsafe { alsa::snd_seq_port_info_set_type(self.0, c.bits() as c_uint) }
     }
 
+    pub fn address(&self) -> Addr {
```

Comment:
Shorten address to "addr" and add a comment - functions that do not represent a 1:1 relationship with an alsa-lib function should have a comment.

Reply:
Shortened and commented!